protostar/io/simple_table.py

```diff
@@ -0,0 +1,10 @@
+def format_as_table(data: dict) -> list[str]:
+    max_key_length = 0
+    for key in data:
+        key_str = str(key)
+        if len(key_str) > max_key_length:
+            max_key_length = len(key_str)
```

Comment:
You can use `max()`

Reply:
[use max](https://github.com/software-mansion/protostar/pull/1224/commits/0ee9855428fccf9022a7a44d1a16820b0fec7eb4)